packages/components/.storybook/preview.tsx

```diff
@@ -30,8 +28,6 @@ body,
 #default-layout {
   width: 100%;
   height: 100%;
-  ${({ theme }) => theme.fonts.text}
-  color: ${({ theme }) => theme.colors.neutralD};
```

Comment:
This stuff wasn't passed to DOM at all & when I converted the files to TypeScript, it notified that these variables don't exist.